packages/ui/src/elements/Table/DefaultCell/fields/JSON/index.tsx

```diff
@@ -6,11 +6,13 @@ import React from 'react'
 import './index.scss'
 
 export const JSONCell: React.FC<DefaultCellComponentProps<JSONFieldClient>> = ({ cellData }) => {
-  const textToShow = cellData?.length > 100 ? `${cellData.substring(0, 100)}\u2026` : cellData
+  const cellDataString = JSON.stringify(cellData)
```

Comment:
I think we can expect the incoming `cellData`  to already be in proper JSON format, and not attempt to stringify it here. If the issue is that `cellData` is potentially an array and not a string, then the root issue is likely further up the rendering tree somewhere.

Reply:
This could also lead to future bugs bc this is not done within a `try...catch`.